composer_multi_project_monitoring/main.tf

```diff
@@ -0,0 +1,1018 @@
+# [START composer_multi_project_monitoring]
+#   Monitoring for multiple Cloud Composer environments
+#   
+#   Usage:
+#       1. Create a new project that you will use for monitoring of Cloud Composer environments in other projects
+#       2. Store the name of this project in "monitoring_project" variable below, in the locals block
+#       3. Save the list of projects with Cloud Composer environments to be monitored in "monitored_projects", set in the locals block
+#       4. Run "terraform apply"
+#
+#   The script creates the following resources in the monitoring project:
+#           1. Adds monitored projects to Cloud Monitoring
+#           2. Creates Alert Policies
+#           3. Creates Monitoring Dashboard
```

Comment:
@rogerthatdev is this a common pattern for samples?

Reply:
Are you asking about the comment itself or what the content is describing? :)

Reply:
I was asking Roger if it was okay to have usage info in the sample itself. I hadn't noticed it in other samples but did not see anything in the style guide against either.